AutoRest/Generators/CSharp/Azure.CSharp.Tests/Expected/AcceptanceTests/AzureSpecials/XMsClientRequestIdOperations.cs

```diff
@@ -143,7 +143,12 @@ internal XMsClientRequestIdOperations(AutoRestAzureSpecialParametersTestClient c
             if ((int)_statusCode != 200)
             {
                 var ex = new CloudException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
-                _responseContent = await _httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
+                if (_httpResponse.Content != null) {
```

Comment:
@pscofield Could you see if this change fixes the issue you were seeing? (you can just copy and paste the change in this file into your generated code to see if it works). I'll merge this into master if you no longer get the null reference exception